runelite-client/src/main/java/net/runelite/client/plugins/inventorygrid/InventoryGridPlugin.java

```diff
@@ -46,6 +56,11 @@
 	@Inject
 	private OverlayManager overlayManager;
 
+	@Inject
+	private Client client;
+
+	private Widget bankWidget = null;
```

Comment:
`null` is the default initial value of non-primitive fields, no need to declare it.
```suggestion
	private Widget bankWidget;
```